Content.Client/VendingMachines/UI/VendingMachineMenu.xaml.cs

```diff
@@ -56,10 +51,11 @@ public void Populate(List<VendingMachineInventoryEntry> inventory)
                     icon = SpriteComponent.GetPrototypeIcon(prototype, _resourceCache).Default;
                 }
 
-                _items.AddItem($"{itemName} ({entry.Amount} left)", icon);
+                VendingContents.AddItem($"{itemName} [{entry.Amount}]", icon);
             }
 
-            SetSize = ((longestEntry.Length + 8) * 12, _items.Count * 40 + 50);
+            SetSize = (Math.Clamp((longestEntry.Length + 2) * 12, 300, 400),
+                Math.Clamp(VendingContents.Count * 45, 200, 500));
```

Comment:
If you're gonna do `Resizable="False"` then make it such that there is no empty space, because currently there seems to always be some empty space beneath the items. I don't feel much for the vertical clamping, at least for the lower bound; if a vending machine only has two items it's fine for the window to only be two items high in my opinion.